src/jdk.jartool/share/classes/sun/security/tools/jarsigner/resources/jarsigner.properties

```diff
@@ -207,3 +207,18 @@ Cannot.find.environment.variable.=Cannot find environment variable:\u0020
 Cannot.find.file.=Cannot find file:\u0020
 event.ocsp.check=Contacting OCSP server at %s ...
 event.crl.check=Downloading CRL from %s ...
+manifest.missing.when.reading.jarfile=Manifest is missing when reading via JarFile
+manifest.missing.when.reading.jarinputstream=Manifest is missing when reading via JarInputStream
+manifest.attribute.1.present.when.reading.jarfile.but.missing.via.jarinputstream=Manifest attribute %s is present when reading via JarFile but missing when reading via JarInputStream
+manifest.attribute.1.present.when.reading.jarinputstream.but.missing.via.jarfile=Manifest attribute %s is present when reading via JarInputStream but missing when reading via JarFile
+manifest.attribute.1.differs.jarfile.value.2.jarinputstream.value.3=Manifest attribute %1$s differs: JarFile value = %2$s, JarInputStream value = %3$s
+entry.1.present.when.reading.jarinputstream.but.missing.via.jarfile=Entry %s is present when reading via JarInputStream but missing when reading via JarFile
+entry.1.present.when.reading.jarfile.but.missing.via.jarinputstream=Entry %s is present when reading via JarFile but missing when reading via JarInputStream
+entry.1.present.in.jarfile.but.unreadable=Entry %s is present in JarFile but unreadable
+codesigners.different.for.entry.1.when.reading.jarfile.and.jarinputstream=Code signers are different for entry %s when reading from JarFile and JarInputStream
+entry.1.has.codesigners.in.jarfile.but.not.in.jarinputstream=Entry %s has codesigners in JarFile but not in JarInputStream
+entry.1.has.codesigners.in.jarinputstream.but.not.in.jarfile=Entry %s has codesigners in JarInputStream but not in JarFile
+entries.mismatch.when.comparing.jarfile.and.jarinputstream=Entries mismatch when comparing JarFile and JarInputStream
+jar.contains.internal.inconsistencies.may.result.in.different.contents.when.reading.via.jarfile.and.jarinputstream=This JAR file contains internal inconsistencies that may result in different contents when reading via JarFile and JarInputStream
```

Comment:
Do you think it makes sense to add a ":" at the end of this header line?

Reply:
Yes.